configurations/kafka/kafka-sink-avro.yaml

```diff
@@ -0,0 +1,24 @@
+stress_cmd: >
+  python_thread -thread=KafkaProducerThread -record-format=avro -topic=table1 -num-records=1000
+  -key-schema='{"type":"record","name":"key_schema","fields":[{"name":"id","type":"int"}]}'
+  -value-schema='{"type":"record","name":"value_schema","fields":[{"name":"firstName","type":"string"},{"name":"lastName","type":"string"}]}'
+stress_read_cmd: >
+  python_thread -thread=KafkaValidatorThread -keyspace=keyspace1 -table=table1 -topic=table1 -record-format=avro -batch-size=500
```

Comment:
why batch-size here is 500 and not 1000 like in producer?

Reply:
Changed the code to align with the test config.